src/spyglass/position/v1/position_dlc_centroid.py

```diff
@@ -268,17 +268,26 @@ def make(self, key):
             )
             position = pynwb.behavior.Position()
             velocity = pynwb.behavior.BehavioralTimeSeries()
-            spatial_series = (RawPosition() & key).fetch_nwb()[0][
-                "raw_position"
-            ]
+            if (
```

Comment:
Does this return None if there's no key?

Reply:
if there's no entry in the table it returns an empty object of the table class; the bool of which evaluates to False